skstats/hypotests/calculators/basecalculator.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/python
```

Comment:
This file misses a top-level docstring at least. But best to document the base class, actually :-).

Reply:
I am not sure what you wish here. I can always add a small docstring where I explain for instance the acronyms I use which are now explained with comments `#`

Reply:
At time it is good to start the file with some explanations of what to find in it, see for example https://github.com/scikit-hep/decaylanguage/blob/master/decaylanguage/decay/viewer.py. This is on top of standard code docstrings, of course. This file has none, meaning a help() on anything will provide no hint or help.

See an example from me on classes doc - https://github.com/scikit-hep/decaylanguage/blob/master/decaylanguage/decay/decay.py. Hope you find it good :D.